### PR TITLE
Converts category slug into a macro that can be reused and makes minor README tweaks

### PR DESCRIPTION
**Pull request by himedlooff**

### CODE REVIEW

**Comment by jimmynotjim** [comment]
LGTM! @anselmbradford and @sebworks want to look this over real quick too?

File: README.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ We welcome your feedback and contributions.
 
 ##### Back-end
 - [Sheer](https://github.com/cfpb/sheer)
-- [elasticsearch](http://www.elasticsearch.org/)
+- [elasticsearch](https://www.elasticsearch.org/)
 
 ##### Front-end Build Tools
-- [Node](http://nodejs.org/) and NPM
+- [Node](https://nodejs.org/) and NPM
```
Comment (anselmbradford) [inline README.md]: Technically Node.js if we want to be preCISE.

**Comment by anselmbradford** [comment]
Fixes https://github.com/cfpb/cfgov-refresh/issues/239

**Comment by anselmbradford** [comment]
Can this be merged? cc @jimmynotjim @sebworks

**Comment by sebworks** [comment]
Looks good to me

**Comment by jimmynotjim** [comment]
@anselmbradford I thought this had a merge conflict due to your large readme PR last week? Did that not get merged?

**Comment by jimmynotjim** [comment]
This is where forks make collaboration a bit more difficult. I've resolved all the conflicts but I can't push to @himedlooff's fork. I went a head and published the [resolved branch](https://github.com/cfpb/cfgov-refresh/compare/himedlooff-gh-pages?expand=1) so that we can at least compare the resolved `README.md`. If I can get a :+1: on that here, I'll merge locally and push these changes up to the Bureau's `origin`.

**Comment by anselmbradford** [comment]
:+1:

**Comment by jimmynotjim** [comment]
Merged

**Comment by himedlooff** [comment]
Thanks for fixing those up for me! :smile:

**Pull request merged**